split quantization jit op

lyoka opened the pull request:
Summary: Currently the test_qbatch_norm_relu is containing too many examples and causing timeout. Splitting them for now to fix the timeout issue

Test Plan: buck test caffe2/test:quantization

Differential Revision: D26141037

supriyar commented on test/quantization/test_quantize_jit.py:
```diff
@@ -2041,6 +2055,20 @@ def __init__(self, dim):
 
+        options = itertools.product([True, False], [2, 3])
+        for tracing, dim in options:
+            for instance in [BNFuncRelu(dim)]:
```
Maybe remove this loop?

lyoka replied:
Done

supriyar commented on test/quantization/test_quantize_jit.py:
```diff
@@ -2051,8 +2079,7 @@ def forward(self, x):
         for tracing, dim in options:
-            for instance in [BNRelu(dim, True), BNRelu(dim, False),
-                             BNFuncRelu(dim), BNFuncInplaceRelu(dim)]:
+            for instance in [BNFuncInplaceRelu(dim)]:
```
Same here..

lyoka replied:
Done

lyoka committed:
split quantization jit op (#51329)

Summary:
Pull Request resolved: https://github.com/pytorch/pytorch/pull/51329

Currently the test_qbatch_norm_relu is containing too many examples and causing timeout. Splitting them for now to fix the timeout issue

Test Plan: buck test caffe2/test:quantization

Reviewed By: supriyar

Differential Revision: D26141037

fbshipit-source-id: fb8bbc40c293816b2824ee88b1d611edd70a69e0
